.circleci/config.yml

```diff
@@ -85,6 +85,8 @@ commands:
       - <<: *install_node_modules
       - <<: *persist_cache
       - <<: *attach_to_bootstrap
+      # cypress docker does not support sudo and do not need it but the default node executor does
+      - run: command -v sudo && sudo npm install -g gatsby-dev-cli || npm install -g gatsby-dev-cli
```

Comment:
Let's move this logic into the e2e-test.sh (shell script) so it's not duplicated!